sdk/src/main/java/io/horizen/account/state/ForgerInfoABI.java

```diff
@@ -0,0 +1,42 @@
+package io.horizen.account.state;
```

Comment:
just a note: maybe is better to move the abi descriptors here
/src/main/scala/io/horizen/account/state/nativescdata/forgerstakev2/

Reply:
It must be a java class unfortunately, but I have changed its package so it is under nativescdata.forgerstakev2.